### PR TITLE
New charsheet look.

### PR DESCRIPTION
**Pull request by parvusdomus**

### CODE REVIEW

**Comment by parvusdomus** [comment]
Added small changes to the character sheet

File: lang/en.json [hunk 1]
```diff
@@ -54,10 +54,10 @@
 	"LexArcana.ActionAdd": "Add",
 	"LexArcana.ActionCancel": "Cancel",
 	"LexArcana.Biography": "Biography",
-	"LexArcana.HitPoints": "Hit Points",
+	"LexArcana.HitPoints": "Hit Points | Total Enc.",
```
Comment (pplans) [inline lang/en.json:57]: Wouldn't it be better to add another entry to the Localization ?

File: lang/es.json [hunk 2]
```diff
@@ -54,10 +54,10 @@
 	"LexArcana.ActionAdd": "Añadir",
 	"LexArcana.ActionCancel": "Cancelar",
 	"LexArcana.Biography": "Biografía",
-	"LexArcana.HitPoints": "Puntos Vida",
+	"LexArcana.HitPoints": "Puntos Vida | Carga Total",
```
Comment (pplans) [inline lang/es.json:57]: Same as the english version

File: templates/actors/custos-sheet.html [hunk 3]
```diff
@@ -6,32 +6,39 @@
...
+				<li><div class="portrait">
+                    <img class="laurel" src="./systems/LexArcana/styles/images/laurel1.webp"/>
+                    <img class="profile" src="{{actor.img}}" title="{{actor.name}}" data-edit="img"/>
+                    <img class="laurel" src="./systems/LexArcana/styles/images/laurel2.webp"/>
+                </div></li>
 				<li>
 					<h1 class="charname">
 						<input name="name" type="text" value="{{actor.name}}" placeholder="{{ localize 'LexArcana.Name' }}"/>
 					</h1>
-				</li>
```
Comment (pplans) [inline templates/actors/custos-sheet.html:14]: It seems to me there's a mismatch between <ul> elements and <li> elements can you confirm they all are closed correctly ?

**Review by pplans — COMMENTED** [review]
Some comments need feedback

**Comment by pplans** [comment]
Updated a comment, seems html elements are interpreted

File: templates/actors/custos-sheet.html (hunk 3 above)
Comment (parvusdomus) [reply templates/actors/custos-sheet.html:14]: no, it's ok I just added a new ul inside the first ul it's just bad indentation that makes it look unclosed, sorry my bad.

**Review by pplans — APPROVED** [review]

**Pull request merged**